src/Identity/ApiAuthorization.IdentityServer/src/Configuration/AspNetConventionsConfigureOptions.cs

```diff
@@ -16,6 +16,7 @@ public void Configure(IdentityServerOptions options)
             options.Events.RaiseFailureEvents = true;
             options.Events.RaiseSuccessEvents = true;
             options.Authentication.CookieAuthenticationScheme = IdentityConstants.ApplicationScheme;
+            options.UserInteraction.ErrorUrl = "/Home";
```

Comment:
This was a small bug where errors would go to "Home/Error" this change makes it inline with the default templates.